pkg/awsManager/vpcVpnManager.go

```diff
@@ -401,25 +378,22 @@ func DeleteRouteTables(client clientpkg.Client, vpcId *string) error {
 	return nil
 }
 
-func DeleteNetworkAcl(client clientpkg.Client, vpcId *string) error {
+func deleteNetworkAcl(client clientpkg.Client, vpcId *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file